mac/Makefile

```diff
@@ -13,12 +15,17 @@ libwooting-rgb-sdk.dylib: $(OBJS)
 	$(CC) $(LDFLAGS) $(LIBS) -shared -fPIC -Wl,-install_name,$0.0 $^ -o $@
 
 $(OBJS): %.o: %.c
-	$(CC) $(CFLAGS) -c $(INCLUDES) $< -o $@
+	$(CC) $(CPPFLAGS) $(CFLAGS) -c $(INCLUDES) $< -o $@
 
 clean:
 	rm -f $(OBJS)
 
 install: libwooting-rgb-sdk.dylib
-	cp libwooting-rgb-sdk.dylib /usr/local/lib/libwooting-rgb-sdk.dylib
+	mkdir -p $(prefix)/lib
+	cp libwooting-rgb-sdk.dylib $(prefix)/lib/libwooting-rgb-sdk.dylib
```

Comment:
I have an improvement to make here. We can drop the explicit file name in the destination path, to make things easier to read and less error-prone.

Reply:
I'll push it to this PR.

Reply:
Done.